lib/gitsh/interactive_runner.rb

```diff
@@ -53,6 +58,16 @@ def greeting_enabled?
       env.fetch('gitsh.noGreeting') { 'false' } != 'true'
     end
 
+    def load_gitshrc
+      script_runner.run(gitshrc_path)
+    rescue NoInputError
+      false
```

Comment:
I'd find an empty `rescue` block more clear, actually: you're rescuing and discarding the `NoInputError`.